.internal-ci/docker/support/node_hw/supervisor/conf.d/ledger-distribution.conf

```diff
@@ -0,0 +1,15 @@
+; Copyright (c) 2018-2022 The MobileCoin Foundation
+[program:ledger-distribution]
+priority=50
+; if we don't start in 20 seconds go to fatal
+startsecs=20
+command=/usr/local/bin/wrapper-ledger-distribution.sh 2>&1
```

Comment:
Do we need `2>&1` here, since we're configuring stderr logging below?

Reply:
fixed.